### DISCUSSION TITLE
Daily 1/11

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#266] [General]
Ayer pude terminar de probar todos los controladores junto con el camino mínimo de una subasta desde que se crea la cuenta hasta que se gana una subasta en MySQL, y hoy estuve en la clase de inglés. Ya probé casi todos los queries, quiero terminar con todos ellos hoy. También quiero actualizar el diagrama de clases y hacer un bosquejo de cómo es nuestra estructura de MVC. Con respecto a la evaluación del primer sprint tal vez separe los diagramas del documento donde las pusimos como para que queden más a la vista para que no se pierdan la próxima vez. Con respecto al sitio de wordpress, no puedo hacer que reconozca la versión 7 aunque me dice que ya la tiene instalada así que estoy esperando a que me habiliten un nuevo sitio para Wordpress, espero tenerlo hoy listo. También quiero ponerme a hacer los diagramas de caso de uso que quedan para hacer.

**Comment by AhumadaCeleste** [comment]
Ayer Tuve muchas dificultades con el servicio de internet y en casa, solo pude avanzar un poco con lo de wordpress pero no finalizarlo.
Hoy espero poder hacerlo y proseguir con java. También participare en la clase de SGC a las 19. Mañana quiero hacer algo de Javascript para sumar al proyecto.

**Reply by rpgrca** [reply]
Javascript ;-)

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve viendo videos de casos de uso, me baje un programa para poder hacerlo y estuve practicando un poco. 
Hoy me voy a ocupar de hacer el footer para los html y voy a seguir con el diagrama de casos de uso.
